terra2/assetlist.json

```diff
@@ -135,7 +135,51 @@
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/blue.png"
         }
-      },     
+      }, 
+      {
+        "description": "Liquidity token, NFT, HARVEST FOR VALUE",
+        "type_asset": "cw20",
+        "address": "terra10se906awphtccf4vd83m0ulpmpt9v4msuakmpy0pwvmtxmup3kdq25rayn",
+        "denom_units": [
+          {
+            "denom": "cw20:terra10se906awphtccf4vd83m0ulpmpt9v4msuakmpy0pwvmtxmup3kdq25rayn",
+            "exponent": 0
+          },
+          {
+            "denom": "xxx",
+            "exponent": 10
+          }
+        ],
+        "base": "cw20:terra10se906awphtccf4vd83m0ulpmpt9v4msuakmpy0pwvmtxmup3kdq25rayn",
+        "name": "TheOnlyOne",
+        "display": "xxx",
+        "symbol": "xxx",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/xxx.png"
```

Comment:
Looks like there is an unused xxx3.png. Please remove

Reply:
Looks like you clicked Resolve without resolving the noted issue. Could you please elaborate? Why are you adding xxx3.png?